app/assets/javascripts/components/Glossary.js

```diff
@@ -0,0 +1,85 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+
+  'use strict';
+
+  var Glossary;
+
+  Glossary = function($el, config) {
+    Glossary.baseConstructor.call(this, $el, config);
+
+    var article = $('[data-dough-glossary-content]'),
+        articleContent = $(article).html(),
+        updatedContent;
+
+    $.getJSON('/glossary.json', function(data){
+      data.glossary.map(function (item) {
+
+        // If the article content contains any of the keywords in the JSON
+        if(articleContent.indexOf(item.term) >= 1){
```

Comment:
what about using a function to check if `articleContent.indexOf(item.term) >= 1` here so the comment isnt necessary